docs/content/stable/explore/ysql-language-features/pg-extensions/extension-pgcron.md

```diff
@@ -19,14 +19,14 @@ If the pg_cron leader node fails, another node is automatically elected as the n
 
 ## Set up pg_cron
 
-Before you can use the feature, you must enable it by setting the `enable_pg_cron` flag. To do this, add `enable_pg_cron` to the `allowed_preview_flags_csv` flag and set the `enable_pg_cron` flag to true on all YB-Masters and YB-TServers.
+Before you can use the feature, you must set the `enable_pg_cron` flag to true on all YB-Masters and YB-TServers.
```

Comment:
This will start the pg_cron postgres background worker that is needed for the extension.